Pod/ILGClasses/ILGClasses.h

```diff
@@ -8,6 +8,8 @@
 
 #import <Foundation/Foundation.h>
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
That's.... extraordinarily stupid. :rage: :rage1: :rage2: :rage3: :rage4: :fire: